deploy/helm/sumologic/values.yaml

```diff
@@ -4279,11 +4229,8 @@ metadata:
                   value: kubelet.service
         groupbyattrs/systemd:
           keys:
-            - _SYSTEMD_UNIT
-            - _HOSTNAME
-            - SYSLOG_FACILITY
-            - PRIORITY
```

Comment:
These are used in the `exclude` section of `sourceprocessor`, so we can't just remove them. I replaced them with filter processors here: https://github.com/SumoLogic/sumologic-kubernetes-collection/commit/b9a4875ad6b9e0f2a497db9a41ba52592394dc43

Reply:
I cherry-picked your commit